package.json

```diff
@@ -9,10 +9,9 @@
   "bin": {
     "llnode": "llnode.sh"
   },
-  "//": "(Blame C++)",
   "scripts": {
     "preinstall": "node scripts/configure.js",
-    "install": "./gyp_llnode && ( gmake -C out/ || make -C out/ )",
+    "install": "node-gyp rebuild",
```

Comment:
Is this required? npm will default to running `node-gyp` if it finds a `binding.gyp`.

Reply:
@richardlau It does not run that if there is a `preinstall` hook